ua/org.eclipse.help.base/src/org/eclipse/help/internal/search/SearchIndex.java

```diff
@@ -874,13 +874,18 @@ private void unzipProductIndex() {
 		try (ZipInputStream zis = new ZipInputStream(zipIn)) {
 			ZipEntry zEntry;
 			while ((zEntry = zis.getNextEntry()) != null) {
+				String name = zEntry.getName();
+				File file = new File(destDir, name);
+				if (!file.toPath().normalize().startsWith(destDir.toPath().normalize())) {
+			        throw new RuntimeException("Bad zip entry: "+name); //$NON-NLS-1$
```

Comment:
Formatting is inconsistent due to whitespaces instead of tabs in this line.

Reply:
fixed